test/stickerbook.test.js

```diff
@@ -290,6 +290,32 @@ describe('Stickerbook', () => {
     stickerbook.off('mouse:down', callback);
   });
 
+  it('cannot place a sticker without a position', (done) => {
+    const stickerbook = createStickerbook();
+
+    stickerbook.setSticker('http://www.example.com/images/A.png')
+      .then(() => {
+        try {
+          stickerbook.placeImage({});
+        } catch(e) {
+          return done(); // we want it to err!
+        }
+
+        done('Should have not allowed an empty position');
+      })
+  })
+  
+  it('can place a sticker with a position', (done) => {
+    const stickerbook = createStickerbook();
+
+    stickerbook.setSticker('http://www.example.com/images/A.png')
+      .then(() => {
+        stickerbook.placeSticker({ x: 0, y: 0 });
+        done();
+      })
+      .catch(done);
```

Comment:
this `.catch(done)` - remind me how it works. We don't want this test to pass if the `.placeSticker` function call throws an error. Can we safely remove `catch(done)` here, or is it doing something we need?